ui/components/app/add-network/add-network.js

```diff
@@ -63,28 +69,73 @@ const AddNetwork = ({
           color={COLORS.TEXT_MUTED}
           margin={[4, 0, 3, 0]}
         >
-          {t('customNetworks')}
+          {t('popularCustomNetworks')}
         </Typography>
         {nets.map((item, index) => (
           <Box
             key={index}
             display={DISPLAY.FLEX}
             alignItems={ALIGN_ITEMS.CENTER}
+            justifyContent={JUSTIFY_CONTENT.SPACE_BETWEEN}
             marginBottom={6}
           >
-            <img
-              className="add-network__token-image"
-              src={item?.rpcPrefs?.imageUrl}
-              alt={t('logo', [item.ticker])}
-            />
-            <Typography variant={TYPOGRAPHY.H7} color={COLORS.TEXT_DEFAULT}>
-              {item.ticker}
-            </Typography>
-            <i
-              className="fa fa-plus add-network__add-icon"
-              onClick={onAddNetworkClick}
-              title={`${t('add')} ${item.ticker}`}
-            />
+            <Box display={DISPLAY.FLEX} alignItems={ALIGN_ITEMS.CENTER}>
+              <IconBorder size={24}>
+                <IconWithFallback
+                  icon={item.rpcPrefs.imageUrl}
+                  name={item.nickname}
+                  size={24}
+                />
+              </IconBorder>
+              <Typography
+                variant={TYPOGRAPHY.H7}
+                color={COLORS.TEXT_DEFAULT}
+                fontWeight={FONT_WEIGHT.BOLD}
+                boxProps={{ marginLeft: 2 }}
+              >
+                {item.nickname}
+              </Typography>
+            </Box>
+            <Box display={DISPLAY.FLEX} alignItems={ALIGN_ITEMS.CENTER}>
+              {
+                // Warning for the networks that doesn't use infura.io as the RPC
+                !infuraRegex.test(item.rpcUrl) && (
+                  <Tooltip
+                    className="add-network__warning-tooltip"
+                    position="top"
+                    interactive
+                    html={
+                      <Box margin={3} className="add-network__warning-tooltip">
+                        {t('addNetworkTooltipWarning', [
+                          <a
+                            key="zendesk_page_link"
+                            href="https://metamask.zendesk.com/hc/en-us/articles/4417500466971"
+                            rel="noreferrer"
+                            target="_blank"
+                          >
+                            {t('learnMoreUpperCase')}
+                          </a>,
+                        ])}
+                      </Box>
+                    }
+                    trigger="mouseenter"
+                    theme="light"
```

Comment:
Remove this so tooltip will work in dark mode

```suggestion
```

### Before
<img width="1073" alt="Screen Shot 2022-03-29 at 2 20 02 PM" src="https://user-images.githubusercontent.com/8112138/160709166-a1c8c6cb-f39e-443e-9375-fcd4f5fdacb0.png">


### After

<img width="1077" alt="Screen Shot 2022-03-29 at 2 20 22 PM" src="https://user-images.githubusercontent.com/8112138/160709183-1bdc1194-95cf-4e08-a57d-2c7def6231b2.png">

Reply:
Hi @georgewrmarshall, When I remove theme="light" form the tooltip and use light mode in the MetaMask storybook tooltip will be dark. Do we have global variable for the dark mode so I make this prop conditional?
<img width="1134" alt="image" src="https://user-images.githubusercontent.com/97883527/160807161-f4130867-c95a-4a14-970a-2e3f84ed1eaa.png">

Reply:
Oh interesting.. Hmm  @darkwing can we get the theme state from redux here?

Reply:
hmm... I don't think we properly support redux in storybook yet

Reply:
@amerkadicE you can probably use the `getTheme` selector from `ui/selectors/selectors.js`

Reply:
Fixed